code/__DEFINES/species_languages.dm

```diff
@@ -17,7 +17,7 @@
 // Languages.
 #define LANGUAGE_COMMON "Galactic Common"
 #define LANGUAGE_CYRILLIC "Techno-Russian"
-#define LANGUAGE_SERBIAN "Serbian"
+#define LANGUAGE_MERC "Merc Geardo"
```

Comment:
Changes like this to a define need to be marked with //Occulus Edit on the line

Reply:
Will make this change! I'll also note what it's replacing in the comment.